src/opencensus/proto/trace/v1/trace_config.proto

```diff
@@ -55,12 +55,15 @@ message ProbabilitySampler {
   double samplingProbability = 1;
 }
 
-// Sampler that makes a constant decision (either always "yes" or always "no")
-// on span sampling.
+// Sampler that always makes a constant decision on span sampling.
 message ConstantSampler {
 
   // Whether spans should be always sampled, or never sampled.
   bool decision = 1;
+
+  // Whether to follow the parent's decision. If there is no parent, use
+  // the decision specified above.
+  bool follow_parent = 2;
```

Comment:
How about this option: just add another bool.

Reply:
I think we don't use this yet, what about we just break compatibility and do the enum version?

Reply:
I'm okay with that.